src/main/java/de/unistuttgart/finitequizbackend/clients/ResultClient.java

```diff
@@ -13,9 +14,9 @@ public interface ResultClient {
     /**
      * Submits the resultDTO to the Overworld-Backend
      *
+     * @param accessToken the users access token
      * @param resultDTO the player submitted result, trimmed down to the data needed for the overworld
      */
     @PostMapping("/submit-game-pass")
-    @Headers("Content-Type: application/json")
```

Comment:
Why did you remove the Content-Type?

I saw it is also not preset in the chickenshock-backend but I think it is better to add it in both instead of remove it

Reply:
My guess: The FeignClient adds it by default when sending an object. But I'm only guessing here.

Reply:
Just copy pasted from crosswordpuzzle backend. I think it is not really needed.